### ISSUE TITLE
 CodeChickenLib-1.7.10-1.1.3.138-universal

### ISSUE THREAD

**Issue by GoldKai** [Chicken-Bones/CodeChickenCore#107]
I don't know how, but the  CodeChickenLib-1.7.10-1.1.3.138-universal that CodeChickenCore auto downloads is not the same as the one that it originally downloaded, and the file that it is downloading now is just causing any modpacks created with it to auto-crash, yet if I copy over and replace the CodeChickenLib-1.7.10-1.1.3.138-universal from an older modpack that I have downloaded the packs start working again.

Please fix this as soon as you can

**Comment by gabenddos** [comment]
Same issue here, please fix!

**Comment by gabenddos** [comment]
Also finally can run the modpack, i downloaded this version of the lib http://chickenbones.net/maven/codechicken/CodeChickenLib/1.7.10-1.1.3.140/CodeChickenLib-1.7.10-1.1.3.140-universal.jar
(1.1.3.140) then renamed the file to 'CodeChickenLib-1.7.10-1.1.3.138-universal' inside /modes/1.7.10 folder and it worked!
Huge thanks @GoldKai for the lead! I couldn't fix it otherwise!

**Comment by LMAA-basedow** [comment]
It doesnt work because the Core Mod tries to download

http://files.minecraftforge.net/maven/codechicken/CodeChickenLib/1.7.10-1.1.3.138/CodeChickenLib-1.7.10-1.1.3.138-universal.jar

wich doesnt exist (anymore?). This leads to a .jar-file being created with size 1KB wich obviously is not correct...

So maybe you have to make the file available again or change the dependencies.info in the Core mod to download CodeChickenLib-1.7.10-1.1.3.140-universal.jar like this:

"repo": "http://files.minecraftforge.net/maven/codechicken/CodeChickenLib/1.7.10-1.1.3.140/",
"file": "CodeChickenLib-1.7.10-1.1.3.140-universal.jar",

**Comment by MagmaSlime123** [comment]
> I don't know how, but the CodeChickenLib-1.7.10-1.1.3.138-universal that CodeChickenCore auto downloads is not the same as the one that it originally downloaded, and the file that it is downloading now is just causing any modpacks created with it to auto-crash, yet if I copy over and replace the CodeChickenLib-1.7.10-1.1.3.138-universal from an older modpack that I have downloaded the packs start working again.
> 
> Please fix this as soon as you can

To everyone who had this issue, a new version was posted on Curse not long ago with the fixed CCL downloader.

**Comment by rickhenderson** [comment]
> To everyone who had this issue, a new version was posted on Curse not long ago with the fixed CCL downloader.

Do you have a link for this? When I'm using CurseForge / Overwolf, I've tried downloading a new copy of Twilight Forest but it still crashes on CodeChicken.

**Comment by noahthomas2014** [comment]
> > I don't know how, but the CodeChickenLib-1.7.10-1.1.3.138-universal that CodeChickenCore auto downloads is not the same as the one that it originally downloaded, and the file that it is downloading now is just causing any modpacks created with it to auto-crash, yet if I copy over and replace the CodeChickenLib-1.7.10-1.1.3.138-universal from an older modpack that I have downloaded the packs start working again.
> > Please fix this as soon as you can
> 
> To everyone who had this issue, a new version was posted on Curse not long ago with the fixed CCL downloader.

I've searched up CodeChickenLib-1.7.10-1.1.3.138-universal curseforge and they only have 1.8+ codechickenlib, there's no new version unless they decided to take said new V down. A link maybe if you have it?

**Comment by MagmaSlime123** [comment]
> > To everyone who had this issue, a new version was posted on Curse not long ago with the fixed CCL downloader.
> 
> Do you have a link for this? When I'm using CurseForge / Overwolf, I've tried downloading a new copy of Twilight Forest but it still crashes on CodeChicken.

https://www.curseforge.com/minecraft/mc-mods/codechickencore/files/3293859

Make sure you delete the old CCL file when updating.

**Comment by GAMER20785** [comment]
i did that but it still auto downloads it

**Comment by MagmaSlime123** [comment]
> i did that but it still auto downloads it

Make sure your on the latest version and delete the old CCL file.

**Comment by calreveraster** [comment]
I can confirm, seems like it's still happening. I'm getting a 1.1.4.1 now, and it deletes my old CCL files - which I do not want. If I wanted a new version, I'd download it. :P 

Can we just add a config file to turn this off, please?

**Comment by Adrikikicp** [comment]
The same happens on 1.6.4

**Comment by DAOWAce** [comment]
Yeah I'm returning to MC with my old server and both the client and server fail to start because of this missing URL problem.  Setting the file to read-only doesn't avoid the issue.  Disabling the update check in config doesn't avoid the issue.

CodeChicken's stuff seems to just be completely removed from the Forge site as of modern times, and anyone posting on the forums about old versions is met with "we don't support this, use the latest" *thread locked*

The older I get, the more I see the problems with the digital era old people keep complaining about.


**EDIT**: I see CodeChicken has retired from modding back in 1.8.  Only the .48 file on curseforge was updated to a working site.

I guess the only way to get this old stuff working is to self host a file.  If anyone comes here from google like I did seeking a resolution for 1.6.4, throw a file up on a webserver with direct linking and edit the dependencies.info (open the .jar with an archival tool) and change the URL to where your file is hosted.

**Comment by JayFeather2** [comment]
> Also finally can run the modpack, i downloaded this version of the lib http://chickenbones.net/maven/codechicken/CodeChickenLib/1.7.10-1.1.3.140/CodeChickenLib-1.7.10-1.1.3.140-universal.jar (1.1.3.140) then renamed the file to 'CodeChickenLib-1.7.10-1.1.3.138-universal' inside /modes/1.7.10 folder and it worked! Huge thanks @GoldKai for the lead! I couldn't fix it otherwise!

THANKS SO MUCH

**Comment by NetLightDev** [comment]
> Also finally can run the modpack, i downloaded this version of the lib http://chickenbones.net/maven/codechicken/CodeChickenLib/1.7.10-1.1.3.140/CodeChickenLib-1.7.10-1.1.3.140-universal.jar (1.1.3.140) then renamed the file to 'CodeChickenLib-1.7.10-1.1.3.138-universal' inside /modes/1.7.10 folder and it worked! Huge thanks [@GoldKai](https://github.com/GoldKai) for the lead! I couldn't fix it otherwise!

It worked for me, too!
Great thanks !! 👍
